videollama2/model/videollama2_llama.py

```diff
@@ -30,6 +30,10 @@
 class Videollama2Config(LlamaConfig):
     model_type = "videollama2_llama"
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.model_type = "videollama2_llama"
```

Comment:
why set `model_type` again?

Reply:
The `model_type` is a class attribute and will be replaced via arguments. For example, if we use `llama` config to instantiate `videollama2config`, the `model_type` is still `llama` rather than `videollama2`